Yoast/wordpress-seo
# Introduce new graph piece for primary image

## Summary

This PR can be summarized in the following changelog entry:

* Moves the primary image for a page out of the `WebPage` piece into its own graph piece.

## Test instructions
This PR can be tested by following these steps:

* Create a post, add an image in the content.
* See that the image is output int he schema below `WebSite`, in a separate piece.
* See that this image is linked by `@id` in both `WebPage` and `Article`.

## Sample Schema output

```<script type='application/ld+json' class='yoast-schema-graph yoast-schema-graph--main'>{
    "@context": "https://schema.org",
    "@graph": [
        {
            "@type": "Organization",
            "@id": "https://example.com/#organization",
            "name": "Yoast",
            "url": "https://example.com/",
            "sameAs": [
                "https://www.facetest.com/yoast",
                "https://www.linkedin.com/company/yoast-com/",
                "https://en.wikipedia.org/wiki/Yoast",
                "https://twitter.com/yoast"
            ],
            "logo": {
                "@type": "ImageObject",
                "@id": "https://example.com/#logo",
                "url": "https://example.com/wp-content/uploads/2019/03/Yoast_Logo_tagline_Large_RGB.png",
                "width": 520,
                "height": 285,
                "caption": "Yoast"
            },
            "image": {
                "@id": "https://example.com/#logo"
            }
        },
        {
            "@type": "WebSite",
            "@id": "https://example.com/#website",
            "url": "https://example.com/",
            "name": "one.wordpress.test",
            "publisher": {
                "@id": "https://example.com/#organization"
            },
            "potentialAction": {
                "@type": "SearchAction",
                "target": "https://example.com/?s={search_term_string}",
                "query-input": "required name=search_term_string"
            }
        },
        {
            "@type": "ImageObject",
            "@id": "https://example.com/how-to-test/#primaryimage",
            "url": "https://example.com/wp-content/uploads/2019/03/hoodie.jpg",
            "width": 801,
            "height": 801
        },
        {
            "@type": "WebPage",
            "@id": "https://example.com/how-to-test/#webpage",
            "url": "https://example.com/how-to-test/",
            "inLanguage": "en-US",
            "name": "How to test - one.wordpress.test",
            "isPartOf": {
                "@id": "https://example.com/#website"
            },
            "primaryImageOfPage": {
                "@id": "https://example.com/how-to-test/#primaryimage"
            },
            "datePublished": "2019-04-15T12:39:57+00:00",
            "dateModified": "2019-05-24T10:36:43+00:00",
            "breadcrumb": {
                "@id": "https://example.com/how-to-test/#breadcrumb"
            }
        },
        {
            "@type": "BreadcrumbList",
            "@id": "https://example.com/how-to-test/#breadcrumb",
            "itemListElement": [
                {
                    "@type": "ListItem",
                    "position": 1,
                    "item": {
                        "@type": "WebPage",
                        "@id": "https://example.com/",
                        "url": "https://example.com/",
                        "name": "Home"
                    }
                },
                {
                    "@type": "ListItem",
                    "position": 2,
                    "item": {
                        "@type": "WebPage",
                        "@id": "https://example.com/how-to-test/",
                        "url": "https://example.com/how-to-test/",
                        "name": "How to test"
                    }
                }
            ]
        },
        {
            "@type": "Article",
            "@id": "https://example.com/how-to-test/#article",
            "isPartOf": {
                "@id": "https://example.com/how-to-test/#webpage"
            },
            "author": {
                "@id": "https://example.com/author/admin/#author"
            },
            "headline": "How to test",
            "datePublished": "2019-04-15T12:39:57+00:00",
            "dateModified": "2019-05-24T10:36:43+00:00",
            "commentCount": 0,
            "mainEntityOfPage": "https://example.com/how-to-test/#webpage",
            "publisher": {
                "@id": "https://example.com/#organization"
            },
            "image": {
                "@id": "https://example.com/how-to-test/#primaryimage"
            },
            "articleSection": ""
        },
        {
            "@type": [
                "Person"
            ],
            "@id": "https://example.com/author/admin/#author",
            "name": "Joost de Valk",
            "image": {
                "@type": "ImageObject",
                "@id": "https://example.com/#authorlogo",
                "url": "https://example.com/wp-content/uploads/2019/05/Avatar_Joost_500x500.png",
                "width": 500,
                "height": 500,
                "caption": "Joost de Valk"
            },
            "description": "This is my bio",
            "sameAs": [
                "https://www.facebook.com/jdevalk",
                "https://twitter.com/jdevalk"
            ]
        }
    ]
}</script>
```

## Quality assurance


Fixes #13037
Fixes #12992
